### DISCUSSION TITLE
How to access data in a proxy object using Valtio?

### DISCUSSION THREAD

**Discussion by hypo-thesis** [pmndrs/valtio#558] [Q&A]
I am using `Valtio `for my webcam `MediaStream `and yet I end up with a `Proxy {}` instead of the stream as such :

![vbvbvbtgg](https://user-images.githubusercontent.com/28728305/193430702-148ade9d-3aa9-4042-a816-768ee09fb824.jpg)

How can I access `MediaStream `inside the proxy state?

**Comment by dai-shi** [comment]
You should be able to access its properties, but I guess you want to wrap it with `ref()` to avoid proxy.

**Reply by hypo-thesis** [reply]
Ref as in React.useRef?

**Reply by dai-shi** [reply]
Ref from Valtio. 
```js
import { ref } from 'valtio'
```

**Reply by hypo-thesis** [reply]
It did the job smoothly, thanks.
